examples/html-kitchen-sink/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "html-kitchen-sink",
   "version": "4.0.0-alpha.7",
+  "private": true,
   "description": "",
+  "keywords": [],
+  "license": "ISC",
```

Comment:
Is there a reason this is the only subproject with a `ISC` license? From what I could find, [there doesn't seem to be a difference](https://gist.github.com/indexzero/10602128), but just curious why one folder is differently licensed.

Reply:
No there's no reason, the only reason this is in the PR, is because I auto-sort `package.json` files.

Please feel free to change it to MIT.